AmplifyPlugins/Auth/AWSCognitoAuthPlugin/ClientBehavior/AWSCognitoAuthPlugin+ClientBehavior.swift

```diff
@@ -83,6 +83,8 @@ extension AWSCognitoAuthPlugin {
                                 options: AuthSocialWebUISignInOperation.Request.Options?,
                                 listener: AuthSocialWebUISignInOperation.ResultListener?)
         -> AuthSocialWebUISignInOperation {
+        
+        
```

Comment:
nit: vertical whitespace. Please confirm that SwiftFormat/Lint would have caught this. If not, we need to figure out why--either the tool isn't configured right or it's not being invoked on build?
```suggestion
```

Reply:
This is from Auth plugin which was not caught while building datastore plugin. Swift format/lint is working as expected.